src/main/webapp/app/iris/session.service.ts

```diff
@@ -0,0 +1,52 @@
+import { Injectable } from '@angular/core';
+import { IrisStateStore } from 'app/iris/state-store.service';
+import { HttpErrorResponse, HttpResponse } from '@angular/common/http';
+import { ConversationErrorOccurredAction, SessionReceivedAction } from 'app/iris/message-store.model';
+import { IrisHttpSessionService } from 'app/iris/http-session.service';
+import { IrisSession } from 'app/entities/iris/iris-session.model';
+import { IrisHttpMessageService } from 'app/iris/http-message.service';
+import { IrisMessage } from 'app/entities/iris/iris-message.model';
+
+@Injectable()
+export class IrisSessionService {
```

Comment:
Missing documentation here.

Reply:
I don't see that we have documentation for the class. Like here `src/main/webapp/app/core/user/user.service.ts`